docs/content/Examples-Tutorials-Recipes/Calculating-Average-and-Percentiles.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Calculating Average and Percentiles
+permalink: /recipes/percentiles
+category: Examples & Tutorials
+subCategory: Data schema
+menuOrder: 1
+---
+
+## Use case
+
+We want to understand the distribution of values for a certain numeric peroperty
+within a dataset. We're used to average values and intuitively understand how to calculate them. However, we also know that average values can be misleading for
+[skewed](https://en.wikipedia.org/wiki/Skewness) distributions which are frequent in
+the real world: for example, 2.5 is the average value for both `(1, 2, 3, 4)` and
+`(0, 0, 0, 10)`.
+
+So, it's usually better to use [percentiles](https://en.wikipedia.org/wiki/Percentile).
+Parametrized by a fractional number `n = 0..1`, the n-th percentile equals to a value
+that exceeds a specified ratio of values in the distribution.
+The [median](https://en.wikipedia.org/wiki/Median) is a special case: it's defined as
+the 50th percentile (`n = 0.5`), and it can be casually thought of as "the middle"
+value. 2.5 and 0 are the medians of `(1, 2, 3, 4)` and `(0, 0, 0, 10)`, respectively.
+
+## Data schema
+
+Let's explore the data in the `Users` cube that contains various demographic
+information about users, including their age:
+
+```javascript
```

Comment:
```suggestion
```json
```

Reply:
Please let me leave it as JavaScript because the syntax highlighting for JSON just doesn't highlight much :)